### PR TITLE
Corrected the relative path

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Following links are taking to a new page with 403 Error: 

```Starting a Cluster``` , ```Stopping a Cluster``` and ```Interacting with Your Cluster```

This PR fixes the above links. Related Issue #14623

### CODE REVIEW

File: content/en/docs/setup/minikube.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
     Creating machine...
     Starting local Kubernetes cluster...
     ```
-    For more information on starting your cluster on a specific Kubernetes version, VM, or container runtime, see [Starting a Cluster](docs/setup/minikube/#starting-a-cluster).
+    For more information on starting your cluster on a specific Kubernetes version, VM, or container runtime, see [Starting a Cluster](/docs/setup/minikube/#starting-a-cluster).
```
Comment (poothia) [inline content/en/docs/setup/minikube.md]: please replace 
/docs/setup/minikube/#starting-a-cluster 
to
minikube.md#starting-a-cluster

File: content/en/docs/setup/minikube.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
...
 
-2. Now, you can interact with your cluster using kubectl. For more information, see [Interacting with Your Cluster](docs/setup/minikube/#interacting-with-your-cluster).
+2. Now, you can interact with your cluster using kubectl. For more information, see [Interacting with Your Cluster](/docs/setup/minikube/#interacting-with-your-cluster).
```
Comment (poothia) [inline content/en/docs/setup/minikube.md]: please replace 
docs/setup/minikube/#interacting-with-your-cluster
to
minikube.md#interacting-with-your-cluster

File: content/en/docs/setup/minikube.md [hunk 3]
```diff
@@ -144,7 +144,7 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
     Stopping "minikube"...
     "minikube" stopped.
     ```
-	For more information, see [Stopping a Cluster](docs/setup/minikube/#stopping-a-cluster).
+	For more information, see [Stopping a Cluster](/docs/setup/minikube/#stopping-a-cluster).
```
Comment (poothia) [inline content/en/docs/setup/minikube.md]: please replace
/docs/setup/minikube/#stopping-a-cluster
to
minikube.md#stopping-a-cluster

**Comment by hpandeycodeit** [comment]
> please take a look.

@poothia File extension is not needed here. Feel free to check the changes in the staging : 

https://deploy-preview-14656--kubernetes-io-master-staging.netlify.com

**Comment by poothia** [comment]
> > please take a look.
> 
> @poothia File extension is not needed here. Feel free to check the changes in the staging :
> 
> https://deploy-preview-14656--kubernetes-io-master-staging.netlify.com

agreed... but you still need to change it.

you can just change the links as follows:-

docs/setup/minikube/#starting-a-cluster
to
#starting-a-cluster

and likewise.

**Comment by Rajakavitha1** [comment]
@hpandeycodeit Please resolve the conflicts!!!

**Comment by hpandeycodeit** [comment]
@Rajakavitha1 

Looks like this is already fixed as part of this PR: https://github.com/kubernetes/website/pull/14662

Feel free to close this one. 

Thank you!

**Comment by tddorgtfo** [comment]
Fixed #14623

**Pull request closed without merging**